extensions/easy-new-file/src/new-file-here.tsx

```diff
@@ -110,6 +110,23 @@ export default function main() {
                       }}
                     />
                   </ActionPanel.Section>
+                  <ActionPanel.Section title="Edit Template With...">
```

Comment:
You can use the [OpenWith action](https://developers.raycast.com/api-reference/user-interface/actions#action.openwith) for this.

Reply:
Thanks for the reminder! The ones that are not used because I didn't notice such a useful component before.
I've submitted a commit to make the changes。